lib/galaxy/jobs/runners/pulsar.py

```diff
@@ -837,14 +837,21 @@ def _monitor(self):
         self.client_manager.ensure_has_ack_consumers()
 
     def __async_update(self, full_status):
-        job_id = None
+        galaxy_job_id = None
         try:
-            job_id = full_status["job_id"]
-            job, job_wrapper = self.app.job_manager.job_handler.job_queue.job_pair_for_id(job_id)
+            remote_job_id = full_status["job_id"]
+            if len(remote_job_id) == 32:
+                # It is a UUID - assign_ids = uuid in destination params...
+                sa_session = self.app.model.context.current
+                # TODO: Add an index for this column.
```

Comment:
This comment can now we removed I think.

Reply:
Yup... meant to... rebased.